src/pages/views/PageCreate.tsx

```diff
@@ -163,8 +178,13 @@ export const PageCreate: React.FC<PageCreateProps> = ({ params }) => {
               referencePages={searchPagesOpts.data?.search.edges.map(
                 edge => edge.node
               )}
+              referenceProducts={searchProductsOpts.data?.search.edges.map(
```

Comment:
`PageDetailsPage` `referenceProducts `  prop type definition expects this to always be an array but undefined is a possible value here. Fallback to an empty array or change the types